src/UI/atoms/Icon.tsx

```diff
@@ -49,35 +49,61 @@ function Icon({
     typescript: IconTypescript,
     webpack: IconWebpack
   };
+
+  const IconComponent = iconComponents[iconName];
```

Comment:
```suggestion
  const IconComponent: SVGIcon = iconComponents[iconName];
```
`SVGIcon` from `__typings__`